micro-rdk-server/esp32/main.rs

```diff
@@ -101,30 +100,35 @@ mod esp32 {
                     .into(),
                 )
                 .expect("Failed to store robot credentials to NVS");
-
-            serve_web(None, repr, max_connection, storage);
         }
-        #[cfg(not(has_robot_config))]
-        {
-            use micro_rdk::common::provisioning::server::ProvisioningInfo;
+
+        let max_connections = unsafe {
+            if !g_spiram_ok {
+                log::info!("spiram not initialized disabling cache feature of the wifi driver");
+                g_wifi_feature_caps &= !(CONFIG_FEATURE_CACHE_TX_BUF_BIT as u64);
+                1
+            } else {
+                3
+            }
+        };
+
+        let info = if cfg!(feature = "provisioning") {
```

Comment:
should the main code in the project template do something similar? (as in pass None if the user says no to including provisioning capabilities)

Reply:
you're right, I sync'd the project template and `micro-rdk-server/esp32/main.rs`